tests/monero_client/monero_cmd.py

```diff
@@ -474,8 +473,8 @@ def validate_prehash_update(self,
 
             navigator.navigate_and_compare(TESTS_ROOT_DIR,
```

Comment:
I think you should replace the navigation by `scenario_navigator` functions, like https://github.com/LedgerHQ/app-ethereum/blob/76e8690a68fb84910d09fbfd14cc975cefabe41a/tests/ragger/test_domain_name.py#L70 ?

Reply:
as you can see the flow for monero are not standard and scenario navigator can not be used easily.